### PR TITLE
Brown 1837

### PR DESCRIPTION
**Pull request by SimonGreenhill**
# Pull request checklist

- [X] add new concept list
- [ ] add new metadata
- [ ] add new Concepticon concept sets
  * [ ] checked whether the new concept(s) can be applied to existing lists with
    `concepticon notlinked --gloss "NEW_GLOSS"`
- [ ] add new Concepticon concept relations
- [ ] refine existing Concepticon concept set mappings
- [ ] refine Concepticon glosses
- [ ] refine Concepticon concept relations
- [ ] refine Concepticon concept definitions
- [ ] retire data

## Additional information

An early comparative and *lexicostatistic* paper. How cool is this from 1837? 

Screen Shot 2023-02-25 at 21 01 46

### CODE REVIEW

**Comment by SimonGreenhill** [comment]
(Also, publishes all his data too. It's been downhill since then :)

**Review by LinguList — COMMENTED** [review]
I can moderate this PR, many thanks, @SimonGreenhill, I'd ask @MuffinLinwist and @mathildavz to have a first review here, and I'd later double-check.

**Review by LinguList — COMMENTED** [review]
Regarding the PDF, @SimonGreenhill, it would be better if you send it directly to @chrzyki so he can upload it to cdstar and will later look into its integration. This needs to be done anyway.

**Comment by chrzyki** [comment]
> Regarding the PDF, @SimonGreenhill, it would be better if you send it directly to @chrzyki so he can upload it to cdstar and will later look into its integration. This needs to be done anyway.

Thanks, I've received the PDF. Will upload as soon as this is merged.

File: concepticondata/references/references.bib [hunk 1]
```diff
@@ -4385,3 +4385,21 @@ @phdthesis{Oliveira2014
   address = {Brasília},
   url     = {https://repositorio.unb.br/handle/10482/17129},
 }
+
+@article{Brown1837,
+  author      = {Brown, Nathan},
+  journal     = {Journal of the Asiatic Society of Bengal (Calcutta)},
+  number      = {2},
+  pages       = {1023-1038},
+  title       = {Comparison of Indo-Chinese languages},
```
Comment (MuffinLinwist) [inline concepticondata/references/references.bib:4394]: I think a capital L is missing for the word _Languages_ in the title.

File: concepticondata/conceptlists.tsv [hunk 2]
```diff
@@ -419,4 +419,5 @@ Wilson-2015-173	Wilson, Fiona M.	2015		173	basic, ranked, norms	Latin	Spanish		W
...
 Oliveira-2014-517	Oliveira, Sanderson Castro Soares de	2022		517	basic	English	Panoan languages	https://repositorio.unb.br/handle/10482/17129	Oliveira2014	Oliveira2014	This list presents the concepts that are reconstructed for Proto-Panoan. In the original data, the reflexes are presented for various Panoan languages.	397-491	
-Hantgan-2022-348	Hantgan, Abbie and Babiker, Hiba and List, Johann-Mattis	2022		348	basic	French	English		Hantgan2022		This concept list is part of a study on the influence on Bangime by neighboring languages.		
\ No newline at end of file
+Hantgan-2022-348	Hantgan, Abbie and Babiker, Hiba and List, Johann-Mattis	2022		348	basic	French	English		Hantgan2022		This concept list is part of a study on the influence on Bangime by neighboring languages.		
+Brown-1837-60	Brown, Nathan	1837		60	basic				Brown1837	Brown1837	An early comparative wordlist designed to enable comparison of the 'Indo-Chinese' languages for scholarly and missionary purposes. The items chosen "are those of the most common objects, and may therefore be regarded the earliest terms in every language, and such as were least liable to be supplanted by foreign words."
```
Comment (MuffinLinwist) [inline concepticondata/conceptlists.tsv:423]: Maybe English could be add as SOURCE_LANGUAGE and Indo-Chinese languages as TARGET_LANGUAGE. 

Also, pages info could be add: 1023-1038

**Review by MuffinLinwist — COMMENTED** [review]
@LinguList and @SimonGreenhill, my review is done. The list seemed to be already finished (according to what I could see from the PDF), so I checked minor details and left two comments.

File: concepticondata/references/references.bib (hunk 1 above)
Comment (LinguList) [reply concepticondata/references/references.bib:4394]: Not necessarily, only if you capitalize the title.

File: concepticondata/conceptlists.tsv (hunk 2 above)
Comment (LinguList) [reply concepticondata/conceptlists.tsv:423]: Yes, that should be done!

File: concepticondata/references/references.bib (hunk 1 above)
Comment (SimonGreenhill) [reply concepticondata/references/references.bib:4394]: the original has it capitalised so I'll enforce that in the  ref.

**Comment by MuffinLinwist** [comment]
Thanks, @SimonGreenhill! All of my requested changes are now fixed.

File: concepticondata/references/references.bib [hunk 3]
```diff
@@ -4385,3 +4385,21 @@ @phdthesis{Oliveira2014
   address = {Brasília},
   url     = {https://repositorio.unb.br/handle/10482/17129},
 }
+
+@article{Brown1837,
+  author      = {Brown, Nathan},
+  journal     = {Journal of the Asiatic Society of Bengal (Calcutta)},
+  number      = {2},
```
Comment (mathildavz) [inline concepticondata/references/references.bib:4392]: Is there a volume you could add?

Comment (LinguList) [reply concepticondata/references/references.bib:4392]: Yep, volume would be useful.

Comment (SimonGreenhill) [reply concepticondata/references/references.bib:4392]: it's there - 3 lines below this one.

**Review by mathildavz — APPROVED** [review]

**Pull request merged**